core/class_test.go

```diff
@@ -40,6 +40,17 @@ func TestClassV1Hash(t *testing.T) {
 	}
 }
 
+func TestCompiledClassHash(t *testing.T) {
+	client := feeder.NewTestClient(t, utils.Integration)
+	gw := adaptfeeder.New(client)
+
+	hash := utils.HexToFelt(t, "0x6d8ede036bb4720e6f348643221d8672bf4f0895622c32c11e57460b3b7dffc")
+	class, err := gw.Class(context.Background(), hash)
+	require.NoError(t, err)
+	got := class.(*core.Cairo1Class).Compiled.Hash()
+	assert.Equal(t, "0x18f95714044fd5408d3bf812bcd249ddec098ab3cd201b7916170cfbfa59e05", got.String())
```

Comment:
Where was this calculated?

Reply:
Hmmm, I'm seeing a different `compiled_class_hash` https://external.integration.starknet.io/feeder_gateway/get_transaction?transactionHash=0x79a7e03b31e0dbc72c3e092cc6459e8ef4e0c26a7221038028736487933c3

`0x57e51d6856a7f80cbe11a9197bf498efa0a8daf4db4d651569c0ba914f7d8ea`